<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="closePlan" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>291</width>
      <height>181</height>
     </rect>
    </property>
    <widget class="QPushButton" name="save">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="name1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Bảng kế Hoạch 
 Đóng Công</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="embryosPlan" native="true">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>50</y>
      <width>291</width>
      <height>181</height>
     </rect>
    </property>
    <widget class="QPushButton" name="save_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="name1_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Bản Kế Hoạch 
 Cấp Phôi</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="inventPlan" native="true">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>50</y>
      <width>291</width>
      <height>181</height>
     </rect>
    </property>
    <widget class="QPushButton" name="save_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="name1_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Bảng kế Hoạch 
 Kiểm Hàng Cho Bếp</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="embryosPlanPart" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>350</y>
      <width>291</width>
      <height>181</height>
     </rect>
    </property>
    <widget class="QPushButton" name="save_4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="name1_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Kế Hoạch Cấp Phôi 
 Bộ Phận </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="productPlan" native="true">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>350</y>
      <width>291</width>
      <height>181</height>
     </rect>
    </property>
    <widget class="QPushButton" name="save_5">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="name1_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Quản Lý Kế Hoạch 
 Sản Xuất</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="productPlanPart" native="true">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>350</y>
      <width>291</width>
      <height>181</height>
     </rect>
    </property>
    <widget class="QPushButton" name="save_6">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="name1_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Kế Hoạch Sản Xuất 
 Cho Tổ, Bộ Phận</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>610</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
